tools/pot/docs/DefaultQuantizationUsage.md

```diff
@@ -11,29 +11,28 @@
+This guide describes how to apply model quantization with the Default Quantization method, without accuracy control using an unannotated dataset. To use this method, you need to create a Python script using an API of Post-Training Optimization Tool (POT), and implement data preparation logic and quantization pipeline. If you are not familiar with Python, you can try [command-line interface](@ref pot_compression_cli_README) of POT which is designed to quantize models from OpenVINO [Model Zoo](https://github.com/openvinotoolkit/open_model_zoo). The figure below shows the common workflow of the quantization script implemented with POT API.
 
 ![](./images/default_quantization_flow.png)
 
 The script should include three basic steps:
-1. Prepare data and dataset interface
-2. Select quantization parameters
-3. Define and run quantization process
+1. Prepare data and dataset interface.
+2. Select quantization parameters.
+3. Define and run quantization process.
 
 ## Prepare data and dataset interface
 In most cases, it is required to implement only `openvino.tools.pot.DataLoader` interface which allows acquiring data from a dataset and applying model-specific pre-processing providing access by index. Any implementation should override the following methods: 
 
-- `__len__()`, returns the size of the dataset
-- `__getitem__()`, provides access to the data by index in range of 0 to `len(self)`. It also can encapsulate the logic of model-specific pre-processing. The method should return data in the following format:
-   - `(data, annotation)`
-
-where `data` is the input that is passed to the model at inference so that it should be properly preprocessed. `data` can be either `numpy.array` object or dictionary, where the key is the name of the model input and value is `numpy.array` which corresponds to this input. Since `annotation` is not used by the Default Quantization method this object can be `None` in this case.
+- The `__len__()`, returns the size of the dataset.
+- The `__getitem__()`, provides access to the data by index in range of 0 to `len(self)`. It also can encapsulate the logic of model-specific pre-processing. This method should return data in `(data, annotation)` format, in which:
+   * The `data` is the input that is passed to the model at inference so that it should be properly preprocessed. It can be either `numpy.array` object or dictionary, where the key is the name of the model input and value is `numpy.array` which corresponds to this input.
```

Comment:
```suggestion
   * The `data` is the input that is passed to the model at inference so that it should be properly preprocessed. It can be either the `numpy.array` object or a dictionary, where the key is the name of the model input and value is `numpy.array` which corresponds to this input.
```